main.js

```diff
@@ -1,7 +1,7 @@
 import {app, BrowserWindow} from 'electron';
 import menubar from 'menubar';
 
-let mb = menubar({maxWidth: 300, maxHeight: 300, minWidth: 300, minHeight: 300, alwaysOnTop: true});
+let mb = menubar({maxWidth: 800, maxHeight: 800, minWidth: 300, minHeight: 300, alwaysOnTop: true});
```

Comment:
still working on getting this to be an interchangeable size but I think this is good for dev.